app/views/participants/show.html.erb

```diff
@@ -6,29 +6,33 @@
     <div class="column">
       <h1>Participant</h1>
     </div>
-    <div class="column column-50">
+    <div class="column column-80">
       <h1><%= @participant.participant_name %></h1>
     </div>
-    
-    <div class="column">
-      <div>
-        <table>
-          <thead>
-            <tr>
-              <th> Session id</th>
-              <th colspan="1"></th>
-            </tr>
-          </thead>
-          <tbody>
-            <tr>
-              <td><%= @participant.session_id %></td>
-              <td><%= link_to 'Destroy', @participant, method: :delete, data: { confirm: 'Are you sure?' } %></td>
-            </tr>
-          </tbody>
-        </table>
+  </div>
+
+  <div class="row">
+      <div class="column">
+        <div>
+          <table>
+            <thead>
+              <tr>
+                <th> Topic</th>
+                <th colspan="1"></th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <% this_session = Session.find(@participant.session_id) %>
+                <td><%= this_session.topic %></td>
+                <td><%= link_to 'Unsubscribe', @participant, method: :delete, data: { confirm: 'Are you sure?' } %></td>
```

Comment:
Nice wording change from "Destroy to Unsubscribe". You can use the associations to pull objects. `@participant.session` should pull up the session for that participant.